test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -150,15 +147,18 @@ func TestSchedule1000Node30KPods(t *testing.T) {
 // 	}
 // }
 
+// testConfig contains the some input parameters needed for running test-suite
 type testConfig struct {
-	numPods                   int
-	numNodes                  int
+	// Note: We don't need numPods, numNodes anymore in this struct but keeping them for backward compatibility
```

Comment:
I don't think it's a good idea, but I don't care about having the comment for now;)